data/remnant.txt

```diff
@@ -8,102 +8,77 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+variant "Starling"
+	"Starling" 8
+	"Starling (Heavy)" 6
+	"Starling (Sniper)" 5
+	"Starling (Hunter)" 4
+	"Starling (Thrasher)" 3
+
+variant "Albatross"
+	"Albatross"
+	"Albatross (Sniper)"
+	"Albatross (Turret)"
+	"Albatross (Heavy)"
+
 fleet "Small Remnant"
 	government "Remnant"
 	names "remnant"
 	cargo 2
 	personality
 		heroic disables plunders appeasing
-	variant 8
-		"Starling"
-	variant 6
-		"Starling (Heavy)"
-	variant 5
-		"Starling (Sniper)"
-	variant 4
-		"Starling (Hunter)"
-	variant 3
-		"Starling (Thrasher)"
+	variant 26
+		variant "Starling"
 	variant 1
-		"Starling (Sniper)" 2
-	variant 1
-		"Albatross"
-	variant 1
-		"Albatross (Sniper)"
-	variant 1
-		"Albatross (Turret)"
-	variant 1
-		"Albatross (Heavy)"
+		variant "Starling" 2
+	variant 4
+		variant "Albatross"
 
 fleet "Large Remnant"
 	government "Remnant"
 	names "remnant"
 	cargo 2
 	personality
 		heroic disables plunders appeasing
-	variant 6
-		"Albatross"
-		"Starling" 2
-	variant 6
-		"Albatross (Sniper)"
-		"Starling (Sniper)" 2
-	variant 6
-		"Albatross (Turret)"
-		"Starling (Thrasher)" 2
-	variant 6
-		"Albatross (Heavy)"
-		"Starling (Heavy)" 2
-	variant 3
-		"Albatross"
-		"Starling (Hunter)" 2
-	variant 3
-		"Albatross (Sniper)"
-		"Starling (Hunter)" 2
+	variant 15
+		variant "Albatross"
+		variant "Starling" 2
 	variant 2
-		"Albatross"
-		"Albatross (Sniper)"
-	variant 2
-		"Albatross (Heavy)"
-		"Albatross (Turret)"
+		variant "Albatross" 2
+
+variant "Gull"
```

Comment:
I also still feel that a different keyword (like versions) would be more appropriate here, since it still seems to have a different meaning than variant within fleets.

Reply:
I'm not against the idea of renaming the GameData (since it's just a matter of changing 3 lines of code, and the functionality is what I desire, so I'm not going to quibble over the name), but I'd be interested to hear if others think that a name change is appropriate.